src/server/graphql/queries/helpers/countTiersForUserId.js

```diff
@@ -0,0 +1,47 @@
+import getRethink from 'server/database/rethinkDriver';
+import {PERSONAL, PRO} from 'universal/utils/constants';
+
+// breaking this out into its own helper so it can be used directly to
+// populate segment traits
+
+const countTiersForUserId = (userId) => {
+  const r = getRethink();
+
+  return r.table('Organization')
+    .getAll(userId, {index: 'orgUsers'})
```

Comment:
Inactive is a trait assigned to a user, and is then denormalized onto the org users. Might be cleaner to just filter out here.